plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/util/MessageConsoleUtilities.java

```diff
@@ -0,0 +1,64 @@
+package com.google.cloud.tools.eclipse.util;
+
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.ui.console.ConsolePlugin;
+import org.eclipse.ui.console.IConsole;
+import org.eclipse.ui.console.IConsoleManager;
+import org.eclipse.ui.console.MessageConsole;
+
+/**
+ * Helper methods for dealing with {@link MessageConsole}s.
+ */
+public class MessageConsoleUtilities {
```

Comment:
I wonder if we should just have a single 'Google Cloud SDK Console' used by deploy and local-devserver.